auth/jwt.go

```diff
@@ -0,0 +1,78 @@
+package auth
```

Comment:
I think it'd be worth adding some happy/sad case tests for this (and the other stuff where it makes sense).